include/sound/sof/topology.h

```diff
@@ -32,6 +32,9 @@ enum sof_comp_type {
 	SOF_COMP_BUFFER,
 	SOF_COMP_EQ_IIR,
 	SOF_COMP_EQ_FIR,
+	SOF_COMP_KEYWORD_DETECT,
+	SOF_COMP_VOICE_DETECT,
+	/* keep FILEREAD/FILEWRITE as the last ones */
 	SOF_COMP_FILEREAD,	/**< host test based file IO */
```

Comment:
you would want to set a value for SOF_COMP_FILEREAD, e.g. 10000 so that it "never" conflicts with regular component ever again?